.nvmrc

```diff
@@ -1 +1 @@
-18.17.1
+18.18.0
```

Comment:
this will break us downstream as as the latest is 18.17.1

Reply:
This should not impact us because this is the version the tests will be run in.